server/graphql/v1/mutations/orders.js

```diff
@@ -286,7 +286,10 @@ export async function createOrder(order, loaders, remoteUser, reqIp) {
         possibleRoles.push(roles.MEMBER);
       }
 
-      if (!remoteUser.hasRole(possibleRoles, order.fromCollective.id)) {
+      if (
+        !remoteUser.hasRole(possibleRoles, order.fromCollective.id) &&
+        fromCollective.CreatedByUserId !== remoteUser.id
```

Comment:
If I create a collective then later get removed from core contributors, I'll still be able to make orders on behalf of the collective I've created. It's a permission issue.

Reply:
While I guess you're not supposed to be removed as an admin of this collective, why can't we just rely on roles here? It's cleaner and easier to maintain. As seen in: https://github.com/opencollective/opencollective-api/pull/2002/files

Reply:
@Betree that’s a great point. While it will never happen in the case of a collective of type USER, this is a very valid argument for organizations and collectives. No reason while you should always be considered as admin. 

I’ll see if can add an ADMIN role in the Members table. I agree with @znarf that it would be the cleanest. If not I’ll restrict this to collectives of type USER.

Reply:
> The Internet generation needs organizations that reflect who we are; where anybody can contribute to a shared mission; **where leaders can easily change**

:slightly_smiling_face: